flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -928,9 +931,16 @@ public void onFatalError(Throwable exception) {
 
 	private class TerminatingFatalErrorHandlerFactory {
 
+		/**
+		 * Create a new {@link TerminatingFatalErrorHandler} for the {@link TaskExecutor} with
+		 * the given index.
+		 *
+		 * @param index into the {{@link #taskManagers}} collection to identify the correct {@link TaskExecutor}.
```

Comment:
Are the double curly braces `{{}}` intended?

Reply:
unintended. will change it.